tools/lint-md/lint-md.src.mjs

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
```

Comment:
nit
```suggestion
import fs from 'fs';
```

Reply:
The line that's there was copy/pasted from our docs. Should it be changed there too?

Reply:
I don't have any strong feelings. I'm suggesting this change because I noticed that `fs` was imported twice in `lint-md.mjs` (https://github.com/nodejs/node/pull/40181/commits/362b0863a3dd0b450283378b03143c1541a22a3a#diff-cb9c80ecc5618f0da4a2ef5648d9f072bc59ef1da6749cec061723a551ea4603R1-R2) I suspected it was because of this line.

Reply:
Changed as you suggested and now the rollup-generated file does not contain the extra import of `fs`. Thanks.